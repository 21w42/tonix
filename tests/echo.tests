echo t
echo --help
help echo
man echo
echo /usr/share
echo /etc/motd
echo \nOverhead the albatross\nHangs motionless upon the air\nAnd deep beneath the rolling waves\nIn labyrinths of coral caves\nAn echo of a distant time\nComes willowing across the sand\nAnd everything is green and submarine\nAnd no one called us to the land\nAnd no one knows the where’s or why’s\nSomething stirs and something tries\nStarts to climb toward the light
echo \n\nStrangers passing in the street\nBy chance, two separate glances meet\nAnd I am you and what I see is me\nAnd do I take you by the hand\nAnd lead you through the land\nAnd help me understand the best I can?\nAnd no one calls us to move on\nAnd no one forces down our eyes\nNo one speaks and no one tries\nNo one flies around the sun
echo \n\nCloudless everyday\nYou fall upon my waking eyes\nInviting and inciting me to rise\nAnd through the window in the wall\nCome streaming in on sunlight wings\nA million bright ambassadors of morning\nAnd no one sings me lullabies\nAnd no one makes me close my eyes\nSo I throw the windows wide\nAnd call to you across the sky
quit

