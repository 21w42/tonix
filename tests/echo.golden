Welcome to Tonix.
Type "help" to get a list of commands.
"man <COMMAND>" or "help <COMMAND>" sometimes might be helpful.
Some options for certain commands work as well.
Feel free to navigate a pre-made file system using intuitive commands.
Path resolution does not work yet, one step at a time please.
Your feedback is highly appreciated!
Have fun :)
$ t

$ Usage: 	echo [OPTION]... [STRING]...

Echo the STRING(s) to standard output.
	-n		do not output the trailing newline
	--help		display this help and exit
	--version	output version information and exit

$ Usage: 	echo [OPTION]... [STRING]...

Echo the STRING(s) to standard output.
	-n		do not output the trailing newline
	--help		display this help and exit
	--version	output version information and exit

$ echo		User Commands
NAME
	echo - display a line of text
SYNOPSIS
	echo [OPTION]... [STRING]...
DESCRIPTION
	Echo the STRING(s) to standard output.
	-n		do not output the trailing newline
	--help		display this help and exit
	--version	output version information and exit

$ /usr/share

$ /etc/motd

$ 
Overhead the albatross
Hangs motionless upon the air
And deep beneath the rolling waves
In labyrinths of coral caves
An echo of a distant time
Comes willowing across the sand
And everything is green and submarine
And no one called us to the land
And no one knows the where’s or why’s
Something stirs and something tries
Starts to climb toward the light

$ 

Strangers passing in the street
By chance, two separate glances meet
And I am you and what I see is me
And do I take you by the hand
And lead you through the land
And help me understand the best I can?
And no one calls us to move on
And no one forces down our eyes
No one speaks and no one tries
No one flies around the sun

$ 

Cloudless everyday
You fall upon my waking eyes
Inviting and inciting me to rise
And through the window in the wall
Come streaming in on sunlight wings
A million bright ambassadors of morning
And no one sings me lullabies
And no one makes me close my eyes
So I throw the windows wide
And call to you across the sky

$ Bye.
